packaging/installer/methods/kickstart.md

```diff
@@ -71,7 +71,7 @@ area](https://learn.netdata.cloud/docs/cloud/spaces#manage-spaces).
 For example:
 
 ```bash
-bash <(curl -Ss https://my-netdata.io/kickstart.sh) --claim-token=TOKEN --claim-rooms=ROOM1,ROOM2
+bash <(curl -Ss https://my-netdata.io/kickstart.sh) --claim-token TOKEN --claim-rooms ROOM1,ROOM2
```

Comment:
```suggestion
bash <(curl -Ss https://my-netdata.io/kickstart.sh) --claim-token TOKEN --claim-rooms ROOM1,ROOM2 --claim-url https://app.netdata.cloud
```

I see that the claim URL is not set by default and it is required

https://github.com/netdata/netdata/blob/78c2af15478ea47e436ed58f0db827e5f37ba26d/packaging/installer/kickstart.sh#L519-L521